tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -427,6 +427,42 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "file-system"
 	},
 
+	/** Firmware layout for the TL-WA850RE v2 */
+	{
+		.id     = "TLWA850REV2",
+		.vendor = "",
+		.support_list =
+			"SupportList:\n"
+			"{product_name:TL-WA850RE,product_ver:2.0.0,special_id:55530000}\n"
+			"{product_name:TL-WA850RE,product_ver:2.0.0,special_id:00000000}\n"
+			"{product_name:TL-WA850RE,product_ver:2.0.0,special_id:55534100}\n",
+		.support_trail = '\x00',
```

Comment:
Hi. You can add here some eu special_ids:
based on latest official EU firmware:
```
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:45550000}
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:4B520000}
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:42520000}
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:4A500000}
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:43410000}
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:41550000}
{product_name:TL-WA850RE,product_ver:2.0.0,special_id:52550000}
```